fairmq/nanomsg/FairMQTransportFactoryNN.cxx

```diff
@@ -63,7 +63,8 @@ FairMQPollerPtr FairMQTransportFactoryNN::CreatePoller(const FairMQSocket& cmdSo
 
 void FairMQTransportFactoryNN::Shutdown()
 {
-    nn_term();
+    // nn_term();
+    // see https://www.freelists.org/post/nanomsg/Getting-rid-of-nn-init-and-nn-term,8
 }
```

Comment:
So it's like shutting down the context, but in nanomsg case without a way to re-create it. We only do this at device exit. I guess the issue is when having multiple devices (or standalone transports) in a process?